dandiapi/api/views/schema.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from dandischema.models import Asset, Dandiset, PublishedAsset, PublishedDandiset
+from dandischema.utils import TransitionalGenerateJsonSchema
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import serializers
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+if TYPE_CHECKING:
+    from rest_framework.request import Request
+
+
+_model_name_mapping = {
+    m.__name__: m
+    for m in [
+        Dandiset,
+        Asset,
+        PublishedDandiset,
+        PublishedAsset,
+    ]
+}
+
+
+class SchemaQuerySerializer(serializers.Serializer):
+    model = serializers.ChoiceField(choices=list(_model_name_mapping))
```

Comment:
```suggestion
    model = serializers.ChoiceField(choices=_model_name_mapping.keys())
```

[`choices` is typed as a `Sequence`](https://github.com/typeddjango/djangorestframework-stubs/blob/master/rest_framework-stubs/fields.pyi#L468), so casting to a list is not necessary.

Reply:
Interesting, I guess the type stubs are wrong... regardless, sorry for the confusion.